packages/components/src/Tooltip/Tooltip.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import { generateSnapshots } from "@chanzuckerberg/story-utils";
+import { userEvent } from "@storybook/testing-library";
+import { composeStories } from "@storybook/testing-react";
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import * as TooltipStoryFile from "./Tooltip.stories";
+
+const { Interactive } = composeStories(TooltipStoryFile);
+
+describe("<Tooltip /> snaps", () => {
+  generateSnapshots(TooltipStoryFile, {
+    getElement: (wrapper) => {
+      return wrapper.baseElement;
+    },
+  });
+});
+describe("<Tooltip />", () => {
+  it("should close tooltip via escape key", async () => {
```

Comment:
You can group these in the same `describe` block:

```suggestion
describe("<Tooltip />", () => {
  generateSnapshots(TooltipStoryFile, {
    getElement: (wrapper) => {
      return wrapper.baseElement;
    },
  });
 
  it("should close tooltip via escape key", async () => {
```

see the `button` spec file for an example

Reply:
Grouped into same `describe` block, thanks